nb-apis/adapters-api/src/main/java/io/nosqlbench/adapters/api/activityimpl/uniform/BaseDriverAdapter.java

```diff
@@ -180,8 +180,10 @@ public NBConfigModel getConfigModel() {
             .add(Param.optional(List.of("workload", "yaml"), String.class, "location of workload yaml file"))
             .add(Param.optional("driver", String.class))
             .add(Param.defaultTo("dryrun", "none").setRegex("(op|jsonnet|emit|none)"))
+            .add(Param.defaultTo("advisor", "none").setRegex("(validate|enforce|none)"))
```

Comment:
There are other commands besides start and run which may need advisor. We might want to hoist this parameter up to NBCLI level.

Reply:
The NBCLI level differs. We should discuss.